charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
+|-----|---------|--------------------------------------------------|-------------|
+| [global.]cluster | `""` |  | Allows to set into the integration the cluster name that is being monitored |
+| [global.]licenseKey | `""` |  | This set this license key to all the products of New Relic |
+| [global.]customSecretName | `""` |  | In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there |
+| [global.]customSecretLicenseKey | `""` |  | In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located |
+| [global.]podLabels | `{}` | yes | A map of labels to add to the pods deployed with this chart |
+| [global.]Labels | `{}` | yes | A map of labels to add to all the objects deployed with this chart |
+| [global.]priorityClassName | `""` |  | Allows to set pod's priorityClassName |
+| [global.]hostNetwork | `false` |  | Allows to set pod's hostNetwork |
+| [global.]dnsConfig | `{}` |  | Allows to set pod's dnsConfig |
+| [global.]image (depends on chart[<sup>2</sup>](#values-managed-globally-2)) | `""` |  | Allow to change the registry where to get the images. Useful when there is an internal image cache/proxy |
+| [global.]image (depends on chart[<sup>2</sup>](#values-managed-globally-2)) | `[]` |  | Allow to set secrets to be able to fetch images |
+| [global.]podSecurityContext | `{}` |  | Allow to set security context (at pod level) |
+| [global.]containerSecurityContext | `{}` |  | Allow to set security context (at container level) |
+| [global.]affinity | `{}` |  | Allows to set pod/node affinity |
+| [global.]nodeSelector | `{}` |  | Allows to schedule the pods to a certain node selector |
+| [global.]tolerations | `[]` |  | Allows to add tolerations to node taints |
+| [global.]serviceAccount.create | `true` |  | Configures if the service account should be created or not |
+| [global.]serviceAccount.name | name of the release |  | Allow to change the name of the service account. This is honored in even it you not create the service account to allow the user to create their own. In case you do not create a service account this defaults to `default` |
+| [global.]serviceAccount.annotations | `{}` | yes | Add these annotations to the service account we create |
+| [global.]customAttributes | `{}` |  | Allows to add extra tags to the cluster and all the metrics emitted to the backend |
+| [global.]fedramp | `false` |  | For the integrations that support it, enables FedRAMP |
+| [global.]lowDataMode | `false` |  | In each integration it has different behaviour but all aims to send less metrics to the backend to try to save costs |
+| [global.]privileged | Depends on the chart[<sup>3</sup>](#values-managed-globally-3) |  | In each integration it has different behaviour but all aims to send less metrics to the backend to try to save costs |
```

Comment:
this was a wrong copy/paste

Reply:
I still have to iterate over the privileged mode. but I solved this issue in [#9f9cfc0](https://github.com/newrelic/nri-kube-events/pull/112/commits/9f9cfc0e51242f18647739df110dcb9b8600a133)